packages/sdk/dappkit/README.md

```diff
@@ -174,15 +177,13 @@ const txObjectExchange = exchange.buy(
 // Later, the amount to be locked will be the parameter `value`.
 const txObjectLock = lockedGold.lock().txo;
 
-// Then we use the 10 CELO to vote for a specific validator group address.
-const validatorGroupAddress = "VALIDATOR_GROUP_ADDRESS";
-const txObjectVote = await election.vote(
+// Then we use the 10 CELO to vote an specific validator group address
+// Here you have to chante the validator group address
+const validatorGroupAddress = "<VALIDATOR_GROUP_ADDRESS>";
+const txObjectVote = (await election.vote(
   validatorGroupAddress, 
-  tenCelo
-).txo;
-
-const dappName = "My DappName";
-const callback = Linking.makeUrl("/my/path");
+  Number(tenCelo)
```

Comment:
This causes a type error for me since it expects a `BigNumber` i think -- maybe storing the above `tenCelo` as a BigNumber is fine since we don't use the string later anyways?